pkg/bpf/map_linux.go

```diff
@@ -869,22 +837,6 @@ func ConvertKeyValue(bKey []byte, bValue []byte, key interface{}, value interfac
 	return nil
 }
 
-// MetadataDiff compares the metadata of the BPF maps and returns false if the
-// metadata does not match
-func (m *Map) MetadataDiff(other *Map) bool {
```

Comment:
Good spotting, this is replaced by the objCheck() functionality inside of `OpenOrCreateMap()`.

Reply:
:)